src/versions.jl

```diff
@@ -0,0 +1,27 @@
+export YAMLVersion, YAMLV1_1, YAMLV1_2
```

Comment:
Do we want to export these? I imagine that they would be better as strictly internal implementation details and that it would be better to use strings for versions in the user-facing functions.

Reply:
Could you give me some example of user-facing functions? Like this?

```julia
load(str, version="1.1")
```

Reply:
Yes, that's a possibility.